js/views/attachment_view.js

```diff
@@ -198,7 +200,10 @@
         if (!this.objectUrl) {
             this.objectUrl = window.URL.createObjectURL(this.blob);
         }
-        this.view = new View(this.objectUrl, this.model.contentType);
+
+        const {blob} = this;
```

Comment:
If we hold to our new regime of introducing eslint wherever we work, that would include this file. And that would fix the lack of spaces around this. :0)